docs/harfbuzz-sections.txt

```diff
@@ -777,4 +777,13 @@ hb_subset_plan_codepoint_to_old_glyph_mapping
 hb_subset_plan_new_to_old_glyph_mapping
 hb_subset_plan_old_to_new_glyph_mapping
 hb_subset_or_fail
+hb_subset_plan_execute_or_fail
+hb_subset_plan_create_or_fail
+hb_subset_plan_destroy
+hb_subset_plan_old_to_new_glyph_mapping
+hb_subset_plan_new_to_old_glyph_mapping
+hb_subset_plan_unicode_to_old_glyph_mapping
+hb_subset_plan_reference
+hb_subset_plan_set_user_data
+hb_subset_plan_get_user_data
```

Comment:
These entries are already in the file, you only need to rename hb_subset_plan_codepoint_to_old_glyph_mapping.

Reply:
Fixed